src/python-bindings/classad.cpp

```diff
@@ -783,7 +783,8 @@ pythonFunctionTrampoline_internal(const char *name, const classad::ArgumentList&
         pyKw["state"] = wrapper;
     }
 
-    boost::python::object pyResult = py_import("__main__").attr("__builtins__").attr("apply")(pyFunc, pyArgs, pyKw);
+    boost::python::object pyResult = boost::python::eval("lambda f,a,kw: f(*a,**kw)")(pyFunc, pyArgs, pyKw);
```

Comment:
Can you also add a comment here about `apply` being a built-in on python 2 but not python 3, hence the trickery with the lambda?